integrationTests/testProcessorNode.go

```diff
@@ -287,8 +299,7 @@ func (tpn *TestProcessorNode) initTestNode() {
 	)
 	tpn.initRounder()
 	tpn.initStorage()
-	tpn.AccntState, tpn.StateTrie, _ = CreateAccountsDB(factory2.UserAccount)
```

Comment:
tpn.StateTrie was also set when creating the UserAccount. On 
line 286 tpn.AccntState, stateTrie, _ = CreateAccountsDB(factory2.UserAccount) 
the stateTrie is not set. Is it not needed any more?

Reply:
tpn state trie is not more needed as you have it in container.

Reply:
all tries are now inside of trieContainer.

Reply:
ok